presto-iceberg/src/test/java/com/facebook/presto/iceberg/IcebergDistributedSmokeTestBase.java

```diff
@@ -484,33 +488,83 @@ private void testCreateTableLike()
 
         assertUpdate(session, "CREATE TABLE test_create_table_like_copy1 (LIKE test_create_table_like_original)");
         assertEquals(getTablePropertiesString("test_create_table_like_copy1"), "WITH (\n" +
-                "   format = 'PARQUET'\n" +
+                "   format = 'PARQUET',\n" +
+                "   format_version = '1'\n" +
                 ")");
         dropTable(session, "test_create_table_like_copy1");
 
         assertUpdate(session, "CREATE TABLE test_create_table_like_copy2 (LIKE test_create_table_like_original EXCLUDING PROPERTIES)");
         assertEquals(getTablePropertiesString("test_create_table_like_copy2"), "WITH (\n" +
-                "   format = 'PARQUET'\n" +
+                "   format = 'PARQUET',\n" +
+                "   format_version = '1'\n" +
                 ")");
         dropTable(session, "test_create_table_like_copy2");
 
         assertUpdate(session, "CREATE TABLE test_create_table_like_copy3 (LIKE test_create_table_like_original INCLUDING PROPERTIES)");
         assertEquals(getTablePropertiesString("test_create_table_like_copy3"), "WITH (\n" +
                 "   format = 'PARQUET',\n" +
+                "   format_version = '1',\n" +
                 "   partitioning = ARRAY['adate']\n" +
                 ")");
         dropTable(session, "test_create_table_like_copy3");
 
         assertUpdate(session, "CREATE TABLE test_create_table_like_copy4 (LIKE test_create_table_like_original INCLUDING PROPERTIES) WITH (format = 'ORC')");
         assertEquals(getTablePropertiesString("test_create_table_like_copy4"), "WITH (\n" +
                 "   format = 'ORC',\n" +
+                "   format_version = '1',\n" +
                 "   partitioning = ARRAY['adate']\n" +
                 ")");
         dropTable(session, "test_create_table_like_copy4");
 
         dropTable(session, "test_create_table_like_original");
     }
 
+    @Test
+    public void testCreateTableWithFormatVersion()
+    {
+        testWithAllFormatVersions(this::testCreateTableWithFormatVersion);
+    }
+
+    private void testCreateTableWithFormatVersion(Session session, String formatVersion)
+    {
+        @Language("SQL") String createTable = "" +
+                "CREATE TABLE test_create_table_with_format_version " +
+                "WITH (" +
+                "format = 'PARQUET', " +
+                "format_version = '" + formatVersion + "'" +
+                ") " +
+                "AS " +
+                "SELECT orderkey AS order_key, shippriority AS ship_priority, orderstatus AS order_status " +
+                "FROM tpch.tiny.orders";
+
+        assertUpdate(session, createTable, "SELECT count(*) from orders");
+
+        String createTableSql = format("" +
```

Comment:
Add "@Language("SQL")"?

Reply:
This annotation was not added here because code editors like IntelliJ are not able to detect the language in the case of format strings. So, even after using this annotation, it doesn't benefit as language injection doesn't take place.